src/main/java/org/mcmonkey/dIRCBot/IRCCommand.java

```diff
@@ -34,22 +34,25 @@
 
     // @Usage
     // Use to connect to an IRC server.
-    // - ~irc connect ircserver@irc.esper.net
+    // - irc connect ircserver@irc.esper.net
     // - irc raw ircserver@irc.esper.net "USER dIRCBot mcmonkey.org mcmonkey.org dIRCBot"
     // - irc raw ircserver@irc.esper.net "NICK dIRCBot"
     // @Usage
     // Use to join a channel within an IRC server.
-    // - ~irc join ircchannel@irc.esper.net#denizen-dev
+    // - irc join ircchannel@irc.esper.net#denizen-dev

```

Comment:
Okay, that makes sense.


On Mon, Jun 29, 2015 at 1:37 PM, mcmonkey <notifications@github.com> wrote:

> In src/main/java/org/mcmonkey/dIRCBot/IRCCommand.java
> <https://github.com/DenizenScript/dIRCBot/pull/2#discussion_r33432381>:
>
> >      // - irc raw ircserver@irc.esper.net "USER dIRCBot mcmonkey.org mcmonkey.org dIRCBot"
> >      // - irc raw ircserver@irc.esper.net "NICK dIRCBot"
> >      // @Usage
> >      // Use to join a channel within an IRC server.
> > -    // - ~irc join ircchannel@irc.esper.net#denizen-dev
> > +    // - irc join ircchannel@irc.esper.net#denizen-dev
>
> The ~ is needed to ensure the channel is joined before you send to it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DenizenScript/dIRCBot/pull/2/files#r33432381>.
>